bfdd/bfdd_nb_config.c

```diff
@@ -356,18 +500,26 @@ int bfdd_bfd_profile_desired_transmission_interval_modify(
 	struct nb_cb_modify_args *args)
 {
 	struct bfd_profile *bp;
+	uint32_t min_tx;
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		min_tx = yang_dnode_get_uint32(args->dnode, NULL);
+		if (min_tx < 10000 || min_tx > 60000000)
+			return NB_ERR_VALIDATION;
 		break;
 
 	case NB_EV_PREPARE:
 		/* NOTHING */
 		break;
 
 	case NB_EV_APPLY:
+		min_tx = yang_dnode_get_uint32(args->dnode, NULL);
 		bp = nb_running_get_entry(args->dnode, NULL, true);
-		bp->min_tx = yang_dnode_get_uint32(args->dnode, NULL);
+		if (bp->min_tx == min_tx)
+			return NB_OK;
+
+		bp->min_tx = min_tx;
 		bfd_profile_update(bp);
```

Comment:
ditto, though I agree with that change.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.